airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -136,13 +136,16 @@ def request_params(
     def chunk_properties(self) -> Iterable[Mapping[str, Any]]:
         selected_properties = self.get_json_schema().get("properties", {})
 
+        def empty_props():
```

Comment:
This should have another naming, how about `include_primary_key()`?

Reply:
renamed